frontend/src/pages/CREResources/AddImagePage/AddNewImageForm.tsx

```diff
@@ -51,6 +52,20 @@ export const AddNewImageForm: React.FC = () => {
       name: false,
     },
   };
+  // default state for "git image" form
+  const defaultGitState: GitImageFormType = {
+    state: {
+      repository: '',
+      name: '',
+      description: '',
+      gitRef: 'master'
```

Comment:
I am not sure if we should set a default value here.

If we do, I would suggest to set it to `main` instead of `master`.

But I would be more inclined towards adding a tooltip that clarifies what this field is about. Something like "You can specify an alternative branch, tag or commit reference within the git repository to build from"